src/jit/codegenarm64.cpp

```diff
@@ -6449,22 +6448,30 @@ CodeGen::genIntrinsic(GenTreePtr treeNode)
 void CodeGen::genPutArgStk(GenTreePtr treeNode)
 {
```

Comment:
This handles passing of hfa structs for GT_CALL nodes.  We need to handle passing of hfa structs for "jmp" il opcode.  That is, changes are needed in genJmpMethod().  At least we should add a NYI if any of the arguments are HFA structs. I suspect we might also need to adding unit tests that cover hfa args.